### ISSUE TITLE
about strip pooling

### ISSUE THREAD

**Issue by swjtulinxi** [Andrew-Qibin/SPNet#4]
hi, can we use this way in the strip pooling that 

your way----
(1)strip pooling-conv2D(3,1)-expand-
(2)strip pooling-conv2D(1,3)-expand-
=fusion((1),(2))--conv2D(1,1)

can we use this ,
(1)strip pooling-expand-
(2)strip pooling-expand-
=fusion((1),(2))--conv2D(3,3)
don't use the conv2D(3,1) and conv2D(1,3) in the middle, instead , use conv2D(3,3) after they fusion??,   i am confused with it ,can you explain it ? thanks.

**Comment by swjtulinxi** [comment]
hi，you said that you use **semseg**.  to train the model.  however ,hou should we choose the right epochs to make sure that our train is the best,  because in semseg,it save the last two .pth  of train epochs,  how do we konw the right epoch,

**Comment by houqb** [comment]
You can do validation during training but it requires more memory.


Kind regards,
Qibin (Andrew)

**Comment by swjtulinxi** [comment]
so，we still can't make sure which epcoh is the best ,however,what we can do is to set a large epcohs,and we can choose a best one throgh doing validation

**Comment by houqb** [comment]
That's correct. Actually, the performance difference between the final
several models is not obvious.

Kind regards,
Qibin (Andrew)

**Comment by swjtulinxi** [comment]
this is not convenient as keras,in keras,it can set the earlystop through monitor

**Comment by swjtulinxi** [comment]
hi，I'm sorry to trouble you again，
![image](https://user-images.githubusercontent.com/50089266/81523487-87b36780-9380-11ea-963b-5ebb8f8688e8.png)
How do you get this kind of picture，because，i did't find the code in your project.

**Comment by swjtulinxi** [comment]
hi,in the paper ,you said that: by default, we add SPMs to the last building block of each stage and all the building blocks of the last stage. **however**,in you code , you just  add spms to the  all the building blocks of the last stage,why ???
spm_on = False
        if planes == 512:
           spm_on = self.spm_on

**Comment by swjtulinxi** [comment]
hi ,i have  a  question about the contextual module  in  Semantic segmentation ,such  as  your MPM, ppm,aspp and so on, Why I use and not use this module has no effect on the results，Does the size of batchsize affect the accuracy of the context module？？
